src/SDKs/DataFactory/Management.DataFactory/Generated/ActivityRunsOperationsExtensions.cs

```diff
@@ -1,15 +1,15 @@
+// <auto-generated>
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for
 // license information.
 //
-// Code generated by Microsoft (R) AutoRest Code Generator.
+// Code generated by Microsoft (R) AutoRest Code Generator 2.2.22.0
```

Comment:
How did you generate the code? I don't get these version numbers when I do it

Reply:
I ran "autorest" that is installed with npm, I believe I have the latest version.

Reply:
how did you invoke it?

Reply:
I ran "autorest".

Reply:
I regenerated SDK based on latest instructions (use .md file as config source)